packages/manager/src/components/TransferDisplay/TransferDisplayUsage.tsx

```diff
@@ -0,0 +1,61 @@
+import Grid from '@mui/material/Unstable_Grid2';
+import * as React from 'react';
+
+import BarPercent from 'src/components/BarPercent';
+import { Typography } from 'src/components/Typography';
+
+import { formatPoolUsagePct } from './utils';
+
+export interface TransferDisplayUsageProps {
+  pullUsagePct: number;
+  quota: number;
+  used: number;
+}
+
+export const TransferDisplayUsage = React.memo(
+  (props: TransferDisplayUsageProps) => {
+    const { pullUsagePct, quota, used } = props;
+    // Don't display usage, quota, or bar percent if the network transfer pool is empty (e.g. account has no resources).
+    const isEmptyPool = quota === 0;
+
+    return (
+      <>
+        <Grid
+          container
+          justifyContent="space-between"
+          spacing={2}
+          sx={{ marginBottom: 0 }}
+        >
+          <Grid style={{ marginRight: 10 }}>
+            {!isEmptyPool ? (
+              <Typography>
+                {used} GB Used ({formatPoolUsagePct(pullUsagePct)})
+              </Typography>
+            ) : (
+              <Typography>
+                Your monthly network transfer will be shown when you create a
+                resource.
```

Comment:
```suggestion
                Your monthly network transfer will be shown when you create a
                resource in this data center.
```

Worth specifying, now that we have DC-specific pools?

Reply:
may be redundant IMO